dia/fakes_analysis.tex

```diff
@@ -0,0 +1,103 @@
+\subsection{Fake Source Injection for DIA}
```

Comment:
I’m not sure if this is the right place to comment, but hopefully someone will get pinged.  Aren’t we supposed to be using “synthetic” instead of “fake”?  Apologies if this has already been addressed!

Reply:
I think RFC-891 is normative, which would suggest just "Source Injection" without either of "fake" or "synthetic."  (RFC-810 was withdrawn.)